test-poc/framework/src/main/java/org/keycloak/test/framework/realm/UserSupplier.java

```diff
@@ -2,6 +2,7 @@
 
 import jakarta.ws.rs.core.Response;
 import org.keycloak.admin.client.resource.UserResource;
+import org.keycloak.representations.idm.RoleRepresentation;
```

Comment:
This import is never used?

Reply:
Is it okay that I fix that in a follow-up? I have another PR that builds on top of this one ;)

Reply:
Yeah sure